Effects/Parallax/source/PlayState.hx

```diff
@@ -0,0 +1,229 @@
+	{
+		// build a gradient sky for the background - make it as big as our screen, and, it's going to be stationary
+		var sky:FlxSprite = FlxGradient.createGradientFlxSprite(FlxG.width, FlxG.height, [0xff6dcff6, 0xff333333], 16);
+		sky.scrollFactor.set();
+		add(sky);
+		
+		
+		var uncoloredMountain:FlxSprite = new FlxSprite(0, 0, "assets/mountains.png");
+		var uncoloredClouds:FlxSprite = new FlxSprite(0, 0, "assets/clouds.png");
+		
+		add(spawnCloud(0)); // add a cloud layer to go behind everything else
+		
+		// we're going to have 6 mountain layers with a cloud layer on top.
+		for (i in 0...6)
+		{
+			
+			add(spawnMountain(i));
+			
+			add(spawnCloud(i));
+			
+		}
+		
+		// this is just a solid-gradient to go behind our ground
+		var _sprSolid = FlxGradient.createGradientFlxSprite(FlxG.width, 64, [0xff333333, 0xff000000], 8);
+		_sprSolid.y = FlxG.height - 64;
+		_sprSolid.scrollFactor.set();
+		add(_sprSolid);		
+		
+		// a tileblock of stuff to go between the player and the mountains
+		var _spookyStuff = new FlxTileblock(0, FlxG.height - 128, Math.ceil(FlxG.width * 4), 64);
+		_spookyStuff.loadTiles("assets/spookystuff.png", 64, 64, 4);
+		_spookyStuff.scrollFactor.set(.95, 0);
+		add(_spookyStuff);		
+		
+		
+		// a tileblock of ground tiles - will move with the player
+		var _ground = new FlxTileblock(0, FlxG.height - 64, FlxG.width * 4, 64);
+		_ground.loadTiles("assets/tiles.png", 16, 16, 1);
+		add(_ground);
+		
+		// our guy for the player to move
+		_guy = new FlxSprite(0, 0, "assets/guy.png");
+		_guy.x = ((FlxG.width * 4) / 2) - (_guy.width / 2);
+		_guy.y = FlxG.height - 72 - _guy.height;
+		_baseY = _guy.y;
+		add(_guy);
+		
+		
+		var txtInst:FlxText = new FlxText(0, FlxG.height - 16, FlxG.width, "Left/Right to Move");
+		txtInst.alignment = FlxTextAlign.CENTER;
+		txtInst.setBorderStyle(FlxTextBorderStyle.SHADOW, 0xff333333, 1, 1);
+		txtInst.scrollFactor.set();
+		add(txtInst);
+		
+		// we're going to have some snow or ash flakes drifting down at different 'levels'. We need a lot of them for the effect to work nicely
+		_flakes = new FlxTypedGroup<Flake>();
+		add(_flakes);
+		
+		for (i in 0...1000)
+		{
+			_flakes.add(new Flake(i % 10));
+		}
+		
+		// Set up our camera to follow the guy, and stay within the confines of our 'world'
+		FlxG.camera.follow(_guy);
+		FlxG.camera.setScrollBounds(0, FlxG.width * 4, 0, FlxG.height);
+		
+		// add some tweens to fade the player in and out and to make him 'float' up and down
+		FlxTween.num(0, 1, 2, { ease:FlxEase.sineInOut, type:FlxTween.PINGPONG }, guyFloat);
+		FlxTween.num(.4, 1, 3, { ease:FlxEase.sineInOut, type:FlxTween.PINGPONG }, guyFade);
+		
+		#if (mobile)
+		_vPad = new FlxVirtualPad(FlxDPadMode.LEFT_RIGHT, FlxActionMode.NONE);
+		
+		add(_vPad);
+		#end
+		
+		super.create();
+	}
+	
+	/**
+	 * This function spawns a new Tileblock of clouds which will be positioned near the top of the screen
+	 */
+	private function spawnCloud(Pos:Int):FlxTileblock
```

Comment:
should be 'pos' instead of 'Pos'. see 'Parameter names' in http://haxeflixel.com/documentation/code-style/